### PR TITLE
add more error docs for queue full (messaging)

### PR DESCRIPTION
**Pull request by dtolb**
## For the Committer

- [x] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [x] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [x] I did these things
- [x] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-add-more-rate-limit-errors.s3-website-us-east-1.amazonaws.com/

File: rateLimits/messageRateLimits.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ All Bandwidth messaging products are rate limited in some fashion.  There are va
 
 ### Sample Message Rate Limit Response
 
+You will receive this error response if:
+
+* Your **burst** rate is exceeded
+* Your **per number** MPS is exceeded (only with queueing disabled)
+
+
```
Comment (jkgibbs) [inline rateLimits/messageRateLimits.md]: A few grammar and clarity changes here:

"Your **burst** rate is exceeded."
"Your **per number** messages per second (MPS) is exceeded (only with queueing disabled)."

File: rateLimits/messageRateLimits.md [hunk 2]
```diff
@@ -68,6 +74,34 @@ Content-Type: application/json
 }
 ```
 
+### Sample Delay Queue full Response
+
+You will receive this error when your message queue is full
```
Comment (jkgibbs) [inline rateLimits/messageRateLimits.md]: Need a colon for consistency here

"You will receive this error when your message queue is full:"

File: rateLimits/messageRateLimits.md [hunk 3]
```diff
@@ -122,7 +156,7 @@ An account has an **Outbound _dequeue_ rate** of 5 MPS, a **Burst _API_ rate** o
 
 ### Back-off and Retry {#backoff-and-retry}
 
-Back-off and Retry adaptively increases delay to attempt to find the quickest possible call pacing without hitting rate limits.  The pseudocode below shows a simple example using a linear coefficient to adjust delay.  Introducing randomness or "jitter" into the delay can help reduce successive collisions.   
+Back-off and Retry adaptively increases delay to attempt to find the quickest possible call pacing without hitting rate limits.  The pseudocode below shows a simple example using a linear coefficient to adjust delay.  Introducing randomness or "jitter" into the delay can help reduce successive collisions.
```
Comment (jkgibbs) [inline rateLimits/messageRateLimits.md]: remove the double spaces after the periods.

File: rateLimits/messageRateLimits.md [hunk 4]
```diff
@@ -161,7 +195,7 @@ WHILE (retry AND (retries < MAX_RETRIES))
 
 ### Queue Management {#queue-management}
 
-Queue Management requires configuring and maintaining a queue of async operations, http API calls for example, facilitating linear control over when the operation is executed.  The pseudocode below shows a simple queueing solution.  For complex queueing tasks, consider using a queue system such as [RabbitMQ](https://www.rabbitmq.com/) or [Mosquitto](https://mosquitto.org/).   
+Queue Management requires configuring and maintaining a queue of async operations, http API calls for example, facilitating linear control over when the operation is executed.  The pseudocode below shows a simple queueing solution.  For complex queueing tasks, consider using a queue system such as [RabbitMQ](https://www.rabbitmq.com/) or [Mosquitto](https://mosquitto.org/).
```
Comment (jkgibbs) [inline rateLimits/messageRateLimits.md]: Please only have one space after periods here.

I know this predated your edits, but I'd like to get that fixed for consistency.

File: rateLimits/messageRateLimits.md (hunk 1 above)
Comment (dtolb) [reply rateLimits/messageRateLimits.md]: done

**Comment by dtolb** [comment]
@jkgibbs & @bwdan tweaked and done.

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-add-more-rate-limit-errors.s3-website-us-east-1.amazonaws.com/

**Pull request merged**